lib/router/model-tasks.js

```diff
@@ -0,0 +1,30 @@
+const { Router } = require('express');
+const Case = require('@ukhomeoffice/taskflow/lib/models/case');
+const { closed } = require('../flow');
+
+module.exports = taskflow => {
+  const router = Router({ mergeParams: true });
+
+  Case.db(taskflow.db);
+
+  router.param('modelId', (req, res, next, modelId) => {
+    req.modelId = modelId;
+    next();
+  });
```

Comment:
It doesn't really _matter_, but as a point of style this `router.param` is pretty much redundant as it's not actually doing anything with the parameter apart from copying it to another part of the request.

I'd generally preserve this kind of code for where there is some kind of parameter validation/processing that can be consolidated across multiple routes that use the parameter. Since there is only one route, and the code doesn't do anything then I'd get rid of it.